adi/assessment_data_import.py

```diff
@@ -147,6 +147,32 @@ def import_data(
 
         # Iterate through assessment data and save each record to the database
         for assessment in assessment_data:
+            # Check for the most required of fields
+            if "id" not in assessment:
+                logging.warning("Assessment missing 'id'! Skipping...")
+                continue
+
+            # Ensure other required fields are present
+            missing_fields = [
+                k
+                for k in (
+                    "Asmt Name",
+                    "Assessment Type",
+                    "created",
+                    "Stakeholder Name",
+                    "status",
+                )
+                if k not in assessment
+            ]
+            if missing_fields:
+                logging.warning(
+                    f"'{assessment['id']}' is missing the following required field(s):"
```

Comment:
I thought we had decided to use the printf-style logging jazz everywhere, since that provides a little bit of optimization.

Reply:
I was keeping it consistent with the rest of the lambda's code. I'm happy to change all the logging to reflect that though.

Reply:
Please see 6c83cda for changing all logging to use the printf-style.